ilcsoft/ilcsoft.py

```diff
@@ -434,6 +434,10 @@ def makeinstall(self):
         # export Python in PATH and LD_LIBRARY_PATH
         f.write( 'export PATH='+py_path+'/bin:${PATH}' + os.linesep  )
         f.write( 'export LD_LIBRARY_PATH='+py_path+'/lib:${LD_LIBRARY_PATH}' + os.linesep  + os.linesep )
+        # copy the PYTHONPATH verbatim from the build setup if it is set
+        python_path = os.getenv('PYTHONPATH')
+        if python_path:
+            f.write('export PYTHONPATH=' + python_path)
```

Comment:
@andresailer would something like this break the nightly builds? Or do they also start from a "clean" environment?